frontend/src/tabs/zoom.pug

```diff
@@ -86,10 +86,11 @@
         a(
           v-if="slice.messageBody !== ''",
           v-on:click="updateExpandedCommitMessagesCount",
-          onclick="toggleNext(this)",
-          title="Click to show/hide the commit message body"
+          onclick="toggleNext(this)"
         )
-          font-awesome-icon.commit-message--button(icon="ellipsis-h")
+          .tooltip
+            font-awesome-icon.commit-message--button(icon="ellipsis-h")
+            span.tooltip-text Click to show/hide the commit message body
```

Comment:
Is it possible to show the message based on whether the commit message body is being shown or hidden? I feel it might be better than a general message.

Reply:
Also, possible to add the same message for the authorship panel?

Reply:
Sure! I can make the message responsive depending on the state.